### PR TITLE
Cleanup attribute declarations

### PR DESCRIPTION
**Pull request by hughbe**
## Proposed Changes
- Cleanup attribute declarations

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/2892)

### CODE REVIEW

**Comment by Tanya-Solyanik** [comment]
@hughbe  - Could you please share what tool you used to format attributes?

**Comment by RussKie** [comment]
> Could you please share what tool you used to format attributes?

I kind of expect an elaborate regex 😏

File: src/System.Windows.Forms/src/System/Windows/Forms/Control.cs [hunk 1]
```diff
@@ -1392,22 +1371,20 @@ public Size ClientSize
...
         /// </summary>
-        [
-            Browsable(false), EditorBrowsable(EditorBrowsableState.Advanced),
-            DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden),
-            Description(nameof(SR.ControlCompanyNameDescr))
-        ]
+        [Browsable(false)]
+        [EditorBrowsable(EditorBrowsableState.Advanced)]
+        [DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden),
+        Description(nameof(SR.ControlCompanyNameDescr))]
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/src/System/Windows/Forms/Control.cs]: These are 2 different attributes, please add brackets

File: src/System.Windows.Forms/src/System/Windows/Forms/Control.cs [hunk 2]
```diff
@@ -1796,13 +1768,11 @@ public virtual Cursor Cursor
...
-            SRCategory(nameof(SR.CatData)),
-            SRDescription(nameof(SR.ControlBindingsDescr)),
-            RefreshProperties(RefreshProperties.All),
-            ParenthesizePropertyName(true)
-        ]
+        [DesignerSerializationVisibility(DesignerSerializationVisibility.Content)]
+        [SRCategory(nameof(SR.CatData))]
+        [SRDescription(nameof(SR.ControlBindingsDescr))]
+        [RefreshProperties(RefreshProperties.All),
+            ParenthesizePropertyName(true)]
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/src/System/Windows/Forms/Control.cs]: please separate these 2 attributes

**Comment by hughbe** [comment]
> @hughbe - Could you please share what tool you used to format attributes?

I used a regex with a bunch of capturing groups and find/replace using VS Code

**Comment by Tanya-Solyanik** [comment]
>I used a regex with a bunch of capturing groups and find/replace using VS Code

How did you double-check that you did not remove any attributes?
You removed some comments, but those are OK.

File: src/System.Windows.Forms/src/System/Windows/Forms/DataGridViewCell.cs [hunk 3]
```diff
@@ -219,10 +209,8 @@ public virtual bool Displayed
             }
         }
 
-        [
-            Browsable(false),
-            EditorBrowsable(EditorBrowsableState.Advanced)
-        ]
+        [Browsable(false)]
+            [EditorBrowsable(EditorBrowsableState.Advanced)]
         public object EditedFormattedValue
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/src/System/Windows/Forms/DataGridViewCell.cs]: Please fix alignment

File: src/System.Windows.Forms/src/System/Windows/Forms/DataGridViewCell.cs [hunk 4]
```diff
@@ -237,10 +225,8 @@ public object EditedFormattedValue
             }
         }
 
-        [
-            Browsable(false),
-            EditorBrowsable(EditorBrowsableState.Advanced)
-        ]
+        [Browsable(false)]
+            [EditorBrowsable(EditorBrowsableState.Advanced)]
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/src/System/Windows/Forms/DataGridViewCell.cs]: please fix alignment

File: src/System.Windows.Forms/src/System/Windows/Forms/DataGridViewCell.cs [hunk 5]
```diff
@@ -565,10 +537,8 @@ internal PropertyStore Properties
             }
         }
 
-        [
-            Browsable(false),
-            DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden)
-        ]
+        [Browsable(false)]
+            [DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden)]
         public virtual bool ReadOnly
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/src/System/Windows/Forms/DataGridViewCell.cs]: please align with the above attribute

File: src/System.Windows.Forms/src/System/Windows/Forms/DataGridViewCell.cs [hunk 6]
```diff
@@ -670,10 +638,8 @@ public virtual bool Resizable
         [Browsable(false)]
         public int RowIndex => OwningRow?.Index ?? -1;
 
-        [
-            Browsable(false),
-            DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden)
-        ]
+        [Browsable(false)]
+            [DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden)]
         public virtual bool Selected
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/src/System/Windows/Forms/DataGridViewCell.cs]: please align

File: src/System.Windows.Forms/src/System/Windows/Forms/DataGridViewCell.cs [hunk 7]
```diff
@@ -831,10 +793,8 @@ public object Tag
             }
         }
 
-        [
-            Browsable(false),
-            DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden)
-        ]
+        [Browsable(false)]
+            [DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden)]
         public string ToolTipText
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/src/System/Windows/Forms/DataGridViewCell.cs]: please indent correctly

File: src/System.Windows.Forms/src/System/Windows/Forms/LinkLabel.cs [hunk 8]
```diff
@@ -2018,12 +2004,10 @@ private void UpdateSelectability()
...
         ///  Determines whether to use compatible text rendering engine (GDI+) or not (GDI).
         /// </summary>
-        [
-        // DefaultValue(false), - // See ShouldSerailizeUseCompatibleTextRendering method.
-        RefreshProperties(RefreshProperties.Repaint),
-        SRCategory(nameof(SR.CatBehavior)),
-        SRDescription(nameof(SR.UseCompatibleTextRenderingDescr))
-        ]
+        [// DefaultValue(false), - // See ShouldSerailizeUseCompatibleTextRendering method.
+        RefreshProperties(RefreshProperties.Repaint)]
```
Comment (RussKie) [inline src/System.Windows.Forms/src/System/Windows/Forms/LinkLabel.cs]: Not correct

**Comment by RussKie** [comment]
> How did you double-check that you did not remove any attributes?

I created a test suite in #2966 to verify any changes to attribute decorations. It is not exhaustive, but should give us a good baseline.
I want #2966 to go first, and then rebase this on top and verify we haven't regressed.

**Comment by RussKie** [comment]
I have run the tests introduced in #2966 against the baselines created in that commit. I am reasonably satisfied this change is cosmetic only.

**Pull request merged**